github_api/misc.py

```diff
@@ -1,5 +1,6 @@
 from math import log
 
+
```

Comment:
@mark-i-m Travis is not 100% reliable ;( there were PRs where he didn't updated statuses at all